lualib/resty/events/protocol.lua

```diff
@@ -83,12 +92,22 @@ function _Server.new()
 
     sock:settimeout(DEFAULT_TIMEOUT)
 
+    local data, err = _recv_frame(sock)
+    if err then
+        return nil, "failed to read worker info: " .. err
+    end
+
+    local info, err = decode(data)
```

Comment:
If we can add a debug message which worker is connecting?

Reply:
@outsinre, I think logging that belongs to `broker`, in this PR I only wanted to change `protocol.lua`.

Reply:
Is this message helpful for debugging?

Reply:
Make sense. If we add this debug, we add to `broker`, not the `protocol`.